client/src/components/Header.js

```diff
@@ -1,32 +1,62 @@
 import React, { Component } from 'react'
 import { Link } from 'react-router-dom'
+import { withRouter } from 'react-router'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 
+import { ExitToApp } from '@material-ui/icons'
```

Comment:
Same as with `Fix/dispatching-application` [information about how to minimize bundle size](https://material-ui.com/guides/minimizing-bundle-size/)